client/my-sites/plans/current-plan/feature-included-card/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button, Card } from '@automattic/components';
+
+import './style.scss';
+
+interface Props {
+	title: string;
+	text: string;
+	illustration: string;
+	showButton: boolean;
+	buttonText?: string;
+	buttonClick?: () => void;
+}
+
+const FeatureIncludedCard: React.FunctionComponent< Props > = ( props ) => {
```

Comment:
Does this type not need to be imported?
And is it even necessary? I would think the following might work:
```suggestion
const FeatureIncludedCard = ( props: Props ) => {
```